util/dir/src/lib.rs

```diff
@@ -226,16 +226,26 @@ impl DatabaseDirectories {
 	}
 }
 
+fn default_path(t: AppDataType, product_name: &'static str) -> Option<PathBuf> {
+	let app_info = AppInfo { name: product_name, author: AUTHOR };
```

Comment:
`product_name` here is always a constant as `AUTHOR`, no need to pass a param

Reply:
Fixed